.github/workflows/test.yml

```diff
@@ -328,6 +329,7 @@ jobs:
           echo "deb [arch=amd64] https://packages.geotrek.fr/ubuntu $(lsb_release -sc) main" | sudo tee /etc/apt/sources.list.d/geotrek.list
           wget -O- "https://packages.geotrek.fr/geotrek.gpg.key" | sudo apt-key add -
           sudo apt-get -q update
+          sudo apt-get -q -y install python3.8 python3.8-venv python3.8-dev
```

Comment:
are you sure is it required ? python3.8 should be in debian depends and python3.8-venv python3.8-dev in build-depends, so install debian package procedure should install them if required